vue-components/src/components/Button.vue

```diff
@@ -188,6 +193,90 @@ $base: '.wikit-Button';
+			}
+
+			&:active {
+				color: $wikit-Button-quiet-neutral-active-color;
+				background-color: $wikit-Button-quiet-neutral-active-background-color;
+				border-color: $wikit-Button-quiet-neutral-active-border-color;
+			}
+
+			// A clicked button is both :active and :focused. Using :not(:active) to avoid mixing the two.
+			&:focus:not(:active) {
+				color: $wikit-Button-quiet-neutral-focus-color;
+				background-color: $wikit-Button-quiet-neutral-focus-background-color;
+				border-color: $wikit-Button-quiet-neutral-focus-border-color;
+				box-shadow: $wikit-Button-quiet-neutral-focus-box-shadow;
+			}
+		}
+		&#{$base}--progressive {
+			color: $wikit-Button-quiet-progressive-color;
+			background-color: $wikit-Button-quiet-background-color;
+			border-color: $wikit-Button-quiet-border-color;
+
+			&:hover {
+				color: $wikit-Button-quiet-progressive-hover-color;
+				background-color: $wikit-Button-quiet-progressive-hover-background-color;
+				border-color: $wikit-Button-quiet-progressive-hover-border-color;
+			}
+
+			&:active {
+				color: $wikit-Button-quiet-progressive-active-color;
+				background-color: $wikit-Button-quiet-progressive-active-background-color;
+				border-color: $wikit-Button-quiet-progressive-active-border-color;
+			}
+
+			// A clicked button is both :active and :focused. Using :not(:active) to avoid mixing the two.
+			&:focus:not(:active) {
+				color: $wikit-Button-quiet-progressive-focus-color;
+				background-color: $wikit-Button-quiet-progressive-focus-background-color;
+				border-color: $wikit-Button-quiet-progressive-focus-border-color;
+				box-shadow: $wikit-Button-quiet-progressive-focus-box-shadow;
+			}
+		}
+		&#{$base}--destructive {
+			color: $wikit-Button-quiet-destructive-color;
+			background-color: $wikit-Button-quiet-background-color;
+			border-color: $wikit-Button-quiet-border-color;
+
+			&:hover {
+				color: $wikit-Button-quiet-destructive-hover-color;
+				background-color: $wikit-Button-quiet-destructive-hover-background-color;
+				border-color: $wikit-Button-quiet-destructive-hover-border-color;
+			}
+
+			&:active {
+				color: $wikit-Button-quiet-destructive-active-color;
+				background-color: $wikit-Button-quiet-destructive-active-background-color;
+				border-color: $wikit-Button-quiet-destructive-active-border-color;
+			}
+
+			// A clicked button is both :active and :focused. Using :not(:active) to avoid mixing the two.
+			&:focus:not(:active) {
+				color: $wikit-Button-quiet-destructive-focus-color;
+				background-color: $wikit-Button-quiet-destructive-focus-background-color;
+				border-color: $wikit-Button-quiet-destructive-focus-border-color;
+				box-shadow: $wikit-Button-quiet-destructive-focus-box-shadow;
+			}
+		}
+	}
+	&#{$base}--quiet:disabled {
+		color: $wikit-Button-quiet-disabled-color;
+		background-color: $wikit-Button-quiet-background-color;
+		border-color: $wikit-Button-quiet-border-color;
+		cursor: default;
```

Comment:
```suggestion
		cursor: default;
		pointer-events: none;
```

The "disabled" quiet button still displays visual effects on hover and click. With `pointer-events: none;` any mouse interaction is ignored. (keyboard interactions are already ignored because it is disabled and therefore not focusable)